core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1234,15 +1235,8 @@ public int getExpectedPort() {
             return Jenkins.getSlaveAgentPortInitialValue(slaveAgentPort);
         }
 
-        public String getExpectedPortString() {
-            return Integer.toString(getExpectedPort());
-        }
-
-        /**
-         * Depending on whether the user said "yes" or "no", send him to the right place.
-         */
         public void doAct(StaplerRequest req, StaplerResponse rsp) throws IOException {
-            if (req.hasParameter("fix")) {
+            if (req.hasParameter("reset")) {
```

Comment:
:ant: Still weird as there's no way to have a submission that doesn't enter this branch. There's no second button.